src/account/accountsSelectors.js

```diff
@@ -197,12 +197,10 @@ export const getIdentity: GlobalSelector<Identity> = createSelector(getAuth, aut
 );
 
 /**
- * The Zulip server version of the active account.
- *
- * Throws if no active account; undefined if server version not known.
+ * The Zulip server version for this account, or null if unknown.
  *
  * See the `zulipVersion` property of `Account` for details on how this
  * information is kept up to date.
  */
-export const getServerVersion = (state: GlobalState): ZulipVersion | null =>
-  getActiveAccount(state).zulipVersion;
+export const getServerVersion = (state: PerAccountState): ZulipVersion | null =>
```

Comment:
Yes, exactly. At this stage, thunk actions all deal with GlobalState only, getting that from their getState callback. That's why it's helpful that at this stage a GlobalState can still be used implicitly as a PerAccountState -- it lets them pass those GlobalState values to the per-account selectors they use, without having to insert explicit conversions everywhere (which would be undesirable because the bulk of them are in thunk actions that are themselves per-account, so when the migration is complete they'll have `getState: () => PerAccountState` and we'd be taking those explicit conversions back out again.)

And typingWorker is a helper for some thunk actions, and will get migrated along with those.